src/main/java/envoy/server/net/ConnectionManager.java

```diff
@@ -44,60 +46,74 @@ private ConnectionManager() {}
 
 			// Remove the socket
-			sockets.entrySet().removeIf(e -> e.getValue() == socketId);
+			sockets.entrySet().removeIf(e -> e.getValue() == socketID);
 		}
 	}
 
 	@Override
-	public void socketRegistered(long socketId) { pendingSockets.add(socketId); }
+	public void socketRegistered(long socketID) { pendingSockets.add(socketID); }
 
 	/**
 	 * Associates a socket ID with a user ID.
 	 *
-	 * @param userId   the user ID
-	 * @param socketId the socket ID
+	 * @param userID   the user ID
+	 * @param socketID the socket ID
 	 * @since Envoy Server Standalone v0.1-alpha
 	 */
-	public void registerUser(long userId, long socketId) {
-		sockets.put(userId, socketId);
-		pendingSockets.remove(socketId);
+	public void registerUser(long userID, long socketID) {
+		sockets.put(userID, socketID);
+		pendingSockets.remove(socketID);
 	}
 
 	/**
-	 * @param userId the ID of the user registered at a socket
+	 * @param userID the ID of the user registered at a socket
 	 * @return the ID of the socket
 	 * @since Envoy Server Standalone v0.1-alpha
 	 */
-	public long getSocketId(long userId) { return sockets.get(userId); }
+	public long getSocketId(long userID) { return sockets.get(userID); }
 
 	/**
-	 * @param socketId the id of the socket whose User is needed
+	 * @param socketID the id of the socket whose User is needed
 	 * @return the userId associated with this socketId
 	 * @since Envoy Server Standalone v0.1-alpha
 	 */
-	public long getUserIdBySocketId(long socketId) {
-		return sockets.entrySet().stream().filter(entry -> entry.getValue().equals(socketId)).findFirst().get().getKey();
+	public long getUserIdBySocketId(long socketID) {
+		return sockets.entrySet().stream().filter(entry -> entry.getValue().equals(socketID)).findFirst().get().getKey();
 	}
 
 	/**
-	 * @param userId the ID of the user to check for
+	 * @param userID the ID of the user to check for
 	 * @return {@code true} if the user is online
 	 * @since Envoy Server Standalone v0.1-alpha
 	 */
-	public boolean isOnline(long userId) { return sockets.containsKey(userId); }
+	public boolean isOnline(long userID) { return sockets.containsKey(userID); }
 
 	/**
-	 * @return the userId of all users who are currently online
+	 * @return the userIDs of all users who are currently online
 	 * @since Envoy Server Standalone v0.1-alpha
 	 */
 	public Set<Long> getOnlineUsers() { return sockets.keySet(); }
+
+	/**
+	 * Returns all members of a group who are currently online.
+	 *
+	 * @param group the group to search for
+	 * @return a set of all IDs of currently active members in this group
+	 * @since Envoy Server Standalone v0.1-beta
+	 */
+	public Set<Long> getOnlineUsersOfGroup(Group group) {
+		Set<Long>	onlineMembers	= new HashSet<>();
+		Set<Long>	members			= group.getContacts().stream().map(envoy.server.data.Contact::getID).collect(Collectors.toSet());
+		members.forEach(userID -> { if (isOnline(userID)) onlineMembers.add(userID); });
+		return onlineMembers;
```

Comment:
Please test this before merging!
```suggestion
		return group.getContacts().stream().map(envoy.server.data.Contact::getID).filter(this::isOnline).collect(Collectors.toSet());
```